application/src/main/resources/ecc.yml

```diff
@@ -95,6 +96,10 @@ repair:
   ##
   lock_type: vnode
   ##
+  ## Specifies if tables with TWCS (TimeWindowCompactionStrategy) should be ignored for repair
+  ##
+  ignore_twcs_tables: false
```

Comment:
NIT: Add the parameter towards the end of the file as it is far more niche than alarms and intervals